code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -81,7 +81,7 @@
 	var/obj/item/reagent_containers/glass/G = O
 	if(stat == CONSCIOUS && istype(G) && G.is_open_container())
 		user.visible_message("<span class='notice'>[user] milks [src] using \the [O].</span>")
-		var/transfered = udder.trans_type_to(G, /datum/reagent/drink/milk, rand(5,10))
+		var/transfered = udder.trans_type_to(G, /datum/reagent/drink/milk/goat, rand(5,10))
```

Comment:
You'll want to make sure that the existing cheese recipes still work with these reagents. They should, but give it a try just in case.

Reply:
tiny problem.
goat milk isn't compatible with milkshakes.
possibly cheese as well.
edit: cheese as well.
..
how would I go about making goat milk compatible with milk?

Reply:
fixed. Milk is now global. Will just make a shitload of cultures.